### PR TITLE
feature/validation-forgot

### LINKED ISSUE #5: Validation Forgot
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Validation the email field that is in forgot page.

**Describe alternatives you've considered**
No.

**Additional context**
No.

### PR DESCRIPTION
**Pull request by empix**
# What's the issue linked?

Resolve #5 

# Test cases

Report the test cases that you did in list items:

- [x] Open page
- [x] Validation email field
- [x] Send form

# Did you test this feature on all browsers?

Yes, tested with all the following browsers:

- [x] Firefox
- [x] Chrome

# Description

Currently i'm verifying if the email field is valid if no error shows.
```js 
cy.get("input[name=email] + span").should("not.exist");
```

We need a new page or message to show for says to user that really was sended. Something how a confirmation.

### CODE REVIEW

**Comment by marcobrunodev** [comment]
Greate!

Please, be more careful with your commits. For example, this [commit](https://github.com/CollabCodeTech/training-frontend/pull/16/commits/d40c2e27053d6ad3eb670f4f8810add9afdb4e29) you write, **Execute test of page Signup** but actually you did an **Update test video of the signup page**.

Also, write small commits. For example, this [commit](https://github.com/CollabCodeTech/training-frontend/pull/16/commits/2d5dee77438873c695ce785a13b84a0f01de4009) you can divide into four commits being, one for each command **it** of the test file.

It's important that the commit messages be more expressive. 

The last thing that I recommend, it is careful with the sequence of the commits. For example, the two last commits, the commit **Add validation on FormForgot** need of the file (`validation.js`) add on commit **Add validation for FormForgot**, therefore, is a little happier if you change order.

**Comment by empix** [comment]
> Greate!
> 
> Please, be more careful with your commits. For example, this [commit](https://github.com/CollabCodeTech/training-frontend/pull/16/commits/d40c2e27053d6ad3eb670f4f8810add9afdb4e29) you write, **Execute test of page Signup** but actually you did an **Update test video of the signup page**.
> 
> Also, write small commits. For example, this [commit](https://github.com/CollabCodeTech/training-frontend/pull/16/commits/2d5dee77438873c695ce785a13b84a0f01de4009) you can divide into four commits being, one for each command **it** of the test file.
> 
> It's important that the commit messages be more expressive.
> 
> The last thing that I recommend, it is careful with the sequence of the commits. For example, the two last commits, the commit **Add validation on FormForgot** need of the file (`validation.js`) add on commit **Add validation for FormForgot**, therefore, is a little happier if you change order.


Ok, I will try to make smaller commits and try to split the features more.

File: cypress/integration/pages/Forgot/index.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    cy.get("input[name=email]").type("empix");
+    cy.contains("Enviar").click();
+    cy.contains("Preencha com email válido");
+  });
+
+  it("Send form with all fields valid", function() {
+    cy.visit("/auth/forgot");
+    cy.get("input[name=email]").type("any.email@gmail.com");
+    cy.contains("Enviar").click();
+  });
```
Comment (marcobrunodev) [inline cypress/integration/pages/Forgot/index.test.js]: This test doesn't is verifying if the form will is filled with all the fields valid. It's missing something that has this verification, something that verifies if the user is on the next page or if it appeared a success message.

**Review by marcobrunodev — CHANGES_REQUESTED** [review]

File: cypress/integration/pages/Forgot/index.test.js (hunk 1 above)
Comment (empix) [reply cypress/integration/pages/Forgot/index.test.js]: I forgot. I will change that.
Thanks.

**Review by marcobrunodev — APPROVED** [review]
Nice! Good job!

**Pull request merged**